### PR TITLE
Signup: add promote local business option to signup site goals

### PR DESCRIPTION
**Pull request by yurynix**
image
  
#### Testing instructions
  
1. Run `git checkout add/promote-local-biz-goal` and start your server, or open a [live branch](https://calypso.live/?branch=add/promote-local-biz-goal)
2. Open the [`Signup` page](http://calypso.localhost:3000/start)
3. Check that you can choose `promote local business` as one of your site's goals
  
#### Reviews
  
- [ ] Code
- [ ] Product

### CODE REVIEW

File: client/signup/steps/about/index.jsx [hunk 1]
```diff
@@ -389,6 +389,21 @@ class AboutStep extends Component {
...
+						name="siteGoals"
+						id="promote"
+						onChange={ this.checkBoxHandleChange }
+						defaultChecked={ this.isCheckBoxChecked( 'promote_local_business' ) }
+						value="promote_local_business"
+						className="about__checkbox"
+						onKeyDown={ this.handleCheckboxKeyDown }
+					/>
+					<span className="about__checkbox-label">
+						{ translate( 'Promote your local business' ) }
```
Comment (a8ci18n) [inline client/signup/steps/about/index.jsx]: Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Promote your business or personal brand' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by fditrapani** [comment]
Since this is a work in progress, I'd recommend couple things you might want to add:
- Set a classic site type for this new goal (I'd recommend `page`): https://github.com/Automattic/wp-calypso/blob/master/client/signup/utils.js#L169
- Select a theme for this new goal (I'd recommend `radcliffe-2` because small business is a premium theme): https://github.com/Automattic/wp-calypso/blob/master/client/signup/utils.js#L147

I'd also revisit the copy for your option "Promote your local business" sounds too similar to the previous option. I could see this confusing people.

**Comment by Tug** [comment]
I thought we would delay that change as we were unsure how to add a new option in the list.
The fact that millions of users have already picked "promote your business" and won't be able to see the GMB offer with this approach is problematic.

**Comment by yurynix** [comment]
Sorry @Tug I've missed that discussion, my bad.

**Pull request closed without merging**